README.md

```diff
@@ -843,7 +843,7 @@ Primitive data types refer to basic or fundamental types of data that are built-
 
 #### string
 
-A string is a data type used to represent a sequence of characters. It can include letters, numbers, symbols, and whitespace. In most programming languages, strings are typically enclosed within single quotes ('') or double quotes (""). For example:
+A string is a data type used to represent a sequence of characters. It can include varters, numbers, symbols, and whitespace. In most programming languages, strings are typically enclosed within single quotes ('') or double quotes (""). For example:
```

Comment:
```suggestion
A string is a data type used to represent a sequence of characters. It can include letters, numbers, symbols, and whitespace. In most programming languages, strings are typically enclosed within single quotes ('') or double quotes (""). For example:
```